usr/code.py

```diff
@@ -0,0 +1,4 @@
+move(Direction.UP)
```

Comment:
As previously, IMO this file should stay empty.
What do you think about loading user code from untracked, automatically created `.txt` file?

Reply:
I think that at the moment it doesn't matter.

Reply:
Ok, but FYI I added https://gut-lynx.atlassian.net/browse/LNX-41for the future.